lib/libxdp/xsk.c

```diff
@@ -398,6 +399,38 @@ static enum xdp_attach_mode xsk_convert_xdp_flags(__u32 xdp_flags)
 	return XDP_MODE_NATIVE;
 }
 
+#define MAX_DEV_QUEUE_PATH_LEN 64
+
+static int xsk_get_queues_from_sysfs(const char* ifname, __u32 *rx, __u32 *tx) {
```

Comment:
This should return void since you do not check for the return value anyway. Instead of return err below, just return. rx = 0 or tx = 0 tells the user that this function did not manage to do what it was supposed to do.